src/main/java/com/roadtriprunner/RoadTripRunner/models/AbstractEntity.java

```diff
@@ -1,6 +1,6 @@
 
 package com.roadtriprunner.RoadTripRunner.models;
-import lombok.Data;
+//import lombok.Data;
```

Comment:
can we reinstate this or delete if unnecessary?

Reply:
resolved